lib/routes/newrank/wechat.js

```diff
@@ -1,16 +1,44 @@
 const got = require('@/utils/got');
+const cheerio = require('cheerio');
 const utils = require('./utils');
 const config = require('@/config').value;
 
 module.exports = async (ctx) => {
     const uid = ctx.params.wxid;
     const nonce = utils.random_nonce(9);
+    const { data: summaryHTML } = await got({
+        method: 'get',
+        url: `https://www.newrank.cn/new/readDetial?account=${uid}`,
+        headers: {
+            Connection: 'keep-alive',
+            Cookie: String(config.newrank.cookie),
+        },
+    });
+    let mainsrc = '';
+    const summary$ = cheerio.load(summaryHTML);
+    summary$('script')
+        .toArray()
+        .forEach((item) => {
+            const src = item.attribs.src || '';
+            if (src.startsWith('/new/static/js/main.')) {
+                mainsrc = src;
+            }
+        });
```

Comment:
You can use [.find()](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Array/find)